aiokatcp/sensor.py

```diff
@@ -303,66 +318,85 @@ def close(self) -> None:
         self.observer = None
 
     @abc.abstractmethod
-    def parameters(self) -> tuple:
+    def _parameters(self) -> tuple:
         pass       # pragma: no cover
 
+    def parameters(self) -> tuple:
+        """Return the parameters with which the sensor was created."""
+        if self.is_auto:
+            return (SensorSampler.Strategy.AUTO,)
```

Comment:
Really not sure why we mask the actual strategy and parameters in the case of an AUTO. Underneath this could be pretty much anything, so why not just pass _parameters() back ?

Most likely I am just confused by the layering here :)

Reply:
When the user sets the strategy they're supposed to get the old strategy back, and if they previously set it to `auto` they should get `auto` back.

Reply:
I'll take your word for it :)